admin_tests/nodes/test_views.py

```diff
@@ -277,29 +279,33 @@ def setUp(self):
         self.node = ProjectFactory(creator=self.user)
         self.registration = RegistrationFactory(project=self.node, creator=self.user)
 
-    @mock.patch('website.project.tasks.settings.SHARE_URL', 'ima_real_website')
-    @mock.patch('website.project.tasks.settings.SHARE_API_TOKEN', 'totaly_real_token')
-    @mock.patch('website.project.tasks.send_share_node_data')
-    def test_reindex_node_share(self, mock_update_share):
+    def test_reindex_node_share(self):
         count = AdminLogEntry.objects.count()
         view = NodeReindexShare()
         view = setup_log_view(view, self.request, guid=self.node._id)
-        view.delete(self.request)
+        with mock.patch('api.share.utils.settings.SHARE_ENABLED', True):
+            with mock.patch('api.share.utils.settings.SHARE_API_TOKEN', 'mock-api-token'):
+                with responses.RequestsMock(assert_all_requests_are_fired=True) as rsps:
```

Comment:
not important, but i just happened to learn that in python 3.1+ you can give several context managers to a single `with` instead of nesting multiple:
```py
with mock.patch('api.share.utils.settings.SHARE_ENABLED', True),
    mock.patch('api.share.utils.settings.SHARE_API_TOKEN', 'mock-api-token'),
    responses.RequestsMock(assert_all_requests_are_fired=True) as rsps:
```
...but splitting it into multiple lines like that [might only work in 3.10+](https://bugs.python.org/issue12782)? like i said, not important